src/main/rb/eca.rb

```diff
@@ -41,7 +41,7 @@
 ## Get the project ID from env var, extracting from pattern 'project-###'
 project_id = ENV['GL_REPOSITORY'][8..-1]
 ## Get data about project from API
-project_response = HTTParty.get("http://localhost/api/v4/projects/#{project_id}")
+project_response = HTTParty.get("https://localhost/api/v4/projects/#{project_id}")
```

Comment:
Wouldn't this fail on any non-production instance? Do we maybe want to look at something like an ENV var?

Reply:
It's failing on production which is my main concern right now.

Reply:
I would recommend creating an issue for your recommended enhancement.